crates/driver/src/boundary/settlement.rs

```diff
@@ -224,7 +226,21 @@ fn to_boundary_order(order: &competition::Order) -> Order {
             onchain_order_data: Default::default(),
             is_liquidity_order: order.is_liquidity(),
         },
-        signature: Default::default(),
+        // TODO Different signing schemes imply different sizes of signature data, which indicates
```

Comment:
Correct, ECDSA signatures (EIP-712 and EthSign) both are exactly 65 bytes long, ERC-1271 signatures (sometimes called EIP-1271 - this is because it moved from EIP to ERC recently) have variable length bytes (from 0 to ∞).